test_scripts/Polices/Policy_Table_Update/025_ATF_P_TC_PTU_validation_failure.lua

```diff
@@ -0,0 +1,189 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] PTU validation failure
+-- [HMI API] OnStatusUpdate
+-- [HMI API] OnReceivedPolicyUpdate notification
+--
+-- Description:
```

Comment:
Maybe need to change it to something like
`In case PTU validation fails, SDL must log the error locally and discard the policy table update with No notification of Cloud about invalid data and notify HMI with OnPolicyUpdate(UPDATE_NEEDED) .`